src/absil/ilwrite.fs

```diff
@@ -2698,10 +2627,7 @@ and GetPropertySigAsBytes cenv env prop =
         prop.Args |> List.iter (EmitType cenv env bb))
 
 and GetPropertyAsPropertyRow cenv env (prop:ILPropertyDef) = 
-    let flags = 
-      (if prop.IsSpecialName then 0x0200 else 0x0) ||| 
-      (if prop.IsRTSpecialName then 0x0400 else 0x0) ||| 
-      (if prop.Init <> None then 0x1000 else 0x0)
```

Comment:
Similar to above: ``Init`` is potentially information that has not been folded into ``Attributes``, and would need careful review (and preferably all updates done via a ``ilPropDef.WithInit(v)`` member?)